sklearn/ensemble/_hist_gradient_boosting/tests/test_predictor.py

```diff
@@ -34,8 +34,8 @@ def test_boston_dataset(n_bins):
 
     predictor = grower.make_predictor(bin_thresholds=mapper.bin_thresholds_)
 
-    assert r2_score(y_train, predictor.predict(X_train)) > 0.85
-    assert r2_score(y_test, predictor.predict(X_test)) > 0.70
+    assert r2_score(y_train, predictor.predict(X_train)) > 0.69
+    assert r2_score(y_test, predictor.predict(X_test)) > 0.30
```

Comment:
aren't these rather low? Same with the other PR you have, maybe using another dataset would be more easonale?

Reply:
Yes, that is the problem with the diabetes dataset. I have changed to california housing and it seems to work reasonably well with the original bins.

* train: (bins=200; 0.8233 (bins=256; 0.8340)
* test: (bins=200; 0.8112) (bins=256; 0.8094)

Reply:
The downside of using `fetch_california_housing` is that it requires network access, which means we would need to mark these test with `
@pytest.mark.network`.

Reply:
With some parameter tuning on the diabetes dataset I can get these results:

`n_bins=50`
train: 0.4253613178731953
test: 0.38498296812822475

`n_bins=100`
train: 0.4298426536827863
test: 0.3991035630532065

Parameters:
    min_samples_leaf = 50
    max_leaf_nodes = None

@thomasjpfan and @adrinjalali which dataset do you guys suggest to use?

Reply:
would a `make_regression` with some tuned parameters not be a good option?

Reply:
Good point, I'll try this tomorrow.